addons/l10n_uy/models/l10n_latam_document_type.py

```diff
@@ -15,14 +15,21 @@ def _format_document_number(self, document_number):
             return super()._format_document_number(document_number)
 
         if not document_number:
-            return
+            return False
+
+        if self.code == "0":
+            return document_number
 
         document_number = document_number.strip()
         number_part = re.findall(r'[\d]+', document_number)
         serie_part = re.findall(r'^[A-Za-z]+', document_number)
+        msg = "'%s' " + _("is not a valid value for") + " '%s'. %s"
```

Comment:
Why is this defined here since it's only used in the if below ?
There's no point to make it a variable either.